tcms/testcases/templates/testcases/mutable.html

```diff
@@ -0,0 +1,152 @@
+                </div>
+
+                <label class="col-md-1 col-lg-1">{% trans "Automated" %}</label>
+                <div class="col-md-3 col-lg-3">
+                    <input class="bootstrap-switch" name="is_automated" type="checkbox">
+                </div>
+            </div>
+
+            <div class="form-group">
+                <label class="col-md-1 col-lg-1" for="id_script">{% trans "Script" %}</label>
+                <div class="col-md-3 col-lg-3 {% if form.script.errors %}has-error{% endif %}">
+                    <input type="text" id="id_script" name="script" value="" class="form-control">
+                    {{ form.script.errors }}
+                </div>
+
+                <label class="col-md-1 col-lg-1" for="id_arguments">{% trans "Arguments" %}</label>
+                <div class="col-md-3 col-lg-3 {% if form.arguments.errors %}has-error{% endif %}">
+                    <input type="text" id="id_arguments" name="arguments" value="" class="form-control">
+                    {{ form.arguments.errors }}
+                </div>
+
+                <label class="col-md-1 col-lg-1" for="id_requirments">{% trans "Requirments" %}</label>
+                <div class="col-md-3 col-lg-3 {% if form.requirment.errors %}has-error{% endif %}">
+                    <input type="text" id="id_requirments" name="requirement" value="" class="form-control">
+                    {{ form.requirment.errors }}
+                </div>
+            </div>
+
+            <div class="form-group">
+                <label class="col-md-1 col-lg-1" for="id_reference_link">{% trans "Reference link" %}</label>
+                <div class="col-md-7 col-lg-7 {% if form.extra_link.errors %}has-error{% endif %}">
+                    <input type="text" id="id_reference_link" name="extra_link" value="" class="form-control">
+                    {{ form.extra_link.errors }}
+                </div>
+
+                <label for="id_notes" class="col-md-1 col-lg-1">{% trans "Notes" %}</label>
+                <div class="col-md-3 col-lg-3">
+                    <textarea id="id_notes" rows="4" name="notes" class="form-control"></textarea>
+                    {{ form.notes.errors }}
+                </div>
+            </div>
+
+            <div class="form-group">
+                <div class="col-md-12 col-lg-12">
+                    <label class="{% if form.setup.errors %}has-error{% endif %}">{% trans "Text:" %}</label>
+                    <div>{{ form.text }}</div>
+                    {{ form.text.errors }}
+                </div>
+            </div>
+
+            <button type="submit" class="btn btn-default btn-lg">{% trans "Save" %}</button>
+
+        </form>
+    </div>
+
+<script src="{% static 'bootstrap-select/dist/js/bootstrap-select.min.js' %}"></script>
+<script src="{% static 'bootstrap-switch/dist/js/bootstrap-switch.min.js' %}"></script>
+<script src="{% static "grappelli/jquery/jquery.min.js" %}" type="text/javascript"></script>
```

Comment:
I don't think `text/javascript` does anything useful and this is not consistent across the file. Remove it (also open issue to remove from other files, possibly writing a linter for it).